torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -1082,6 +1100,17 @@ RegisterOperators reg2({
           push(stack, sum);
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::sum.complex(complex[] self) -> complex",
```

Comment:
Is there a test for this? I saw that it was deleted in the latest update.

Reply:
done!